webapp/app/js/perf.js

```diff
@@ -164,11 +166,31 @@ perf.controller('PerfCtrl', [ '$state', '$stateParams', '$scope', '$rootScope',
       $scope.hideTooltip();
     });
 
+    // Highlight the points persisted in the url
+    function highlightDataPoints() {
+      $scope.seriesList.forEach(function(series, i) {
+        if (series.highlighted) {
+          if (series.highlighted.length > 0 && series.visible) {
+            $scope.resetHighlightButton = true;
+            $scope.revisionToHighlight = series.highlighted[1];
+            $scope.plot.highlight(i, series.highlighted[0]);
+          }
+        }
+      });
+    }
+
     function plotGraph() {
       // synchronize series visibility with flot, in case it's changed
       $scope.seriesList.forEach(function(series) {
         series.flotSeries.points.show = series.visible;
       });
+
+      // Show graph-bottom if there's atleast one series
```

Comment:
This code isn't necessary. You can just evaluate whether seriesList.length is greater than 0 directly in the template (see below)